app/views/posts/show.html.erb

```diff
@@ -1 +1,9 @@
 <h1>Specific post</h1>
+<div class='single-post'>
+  <h4>Post #<%= @post.id %> by <%= @post.author.name %></h4>
+  <span>Comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %></span>
```

Comment:
- [ Required ] As requested by the previous reviewer please make sure this `view` has no error, the `has_many` association between the `user` model and `post` model has not been setup correctly which causes an error in the `post-show page` as shown below. Please see the screenshot below and fix this error.

## Show post page error
![Screenshot from 2023-03-09 18-07-37](https://user-images.githubusercontent.com/16392046/224066297-4b2bdcc2-2c0b-4ede-9e14-8730cb96f42c.png)